tools/cli/commands/release.js

```diff
@@ -230,33 +247,80 @@ export async function getReleaseVersion( argv ) {
 		.execSync( `tools/plugin-version.sh ${ argv.project }` )
 		.toString()
 		.trim();
-	potentialVersion = potentialVersion.split( '-' );
-	potentialVersion = potentialVersion[ 0 ].split( '.' ).splice( 0, 2 );
-	potentialVersion = potentialVersion.join( '.' );
+	potentialVersion = potentialVersion.split( '-' ); // e.g., split 10.4-a.8 into [10.4, a.8]
+	let stableVersion = potentialVersion[ 0 ];
+	let alphaVersion = potentialVersion[ 1 ];
+
+	if ( argv.stable || argv.s ) {
+		return stableVersion;
+	}
 
 	// Append '-beta' if necessary.
 	if ( argv.b || argv.beta ) {
-		potentialVersion += '-beta';
+		return `${ stableVersion }-beta`;
 	}
-	argv = await promptForVersion( argv, potentialVersion );
 
-	return argv;
+	// Append '-alpha' if necessary.
+	if ( argv.a || argv[ 'dev-version' ] ) {
+		// Jetpack uses additional versioning for dev/atomic in the form of x.y-a.z
+		if ( argv.project === 'plugins/jetpack' ) {
+			if ( alphaVersion ) {
+				alphaVersion = await getVersionBump( alphaVersion );
+				potentialVersion = `${ stableVersion }-${ alphaVersion }`;
+			} else {
+				stableVersion = await getVersionBump( stableVersion );
+				potentialVersion = `${ stableVersion }-a.0`;
+			}
+		} else {
+			stableVersion = await getVersionBump( stableVersion );
+			potentialVersion = `${ stableVersion }-alpha`;
+		}
+		return potentialVersion;
+	}
 }
 
+/**
+ * Bumps the correct number.
```

Comment:
This does a bump for when composer.json has `.extra.changelogger.versioning` set to 'wordpress'. It gets it wrong for the default 'semver'.

Reply:
Alright, tried to clean this one up as best I could!

Reply:
Looks good to me!